{"uuid": "ffa886b4-26f1-429e-85ad-0e15469e6029", "children": ["3db3b959-b5d7-495b-bf61-b6526da09d7c", "933eb7d0-934e-49f6-9c15-b114a5274cd6", "7a68b351-ade6-47bb-9455-4a3714fdac32"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.\n\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 937, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 790, in call_fixture_func\n    res = next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 760, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 701, in _call_with_optional_argument\n    func(arg)\n  File \"/scipt/python/testerhome_pratice/appium_homework/test_xueqiu_webview.py\", line 25, in setup_class\n    cls.driver.find_element_by_accessibility_id('基金开户').click()\n  File \"/usr/local/lib/python3.7/site-packages/appium/webdriver/webdriver.py\", line 454, in find_element_by_accessibility_id\n    return self.find_element(by=MobileBy.ACCESSIBILITY_ID, value=accessibility_id)\n  File \"/usr/local/lib/python3.7/site-packages/appium/webdriver/webdriver.py\", line 270, in find_element\n    'value': value})['value']\n  File \"/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.7/site-packages/appium/webdriver/errorhandler.py\", line 29, in check_response\n    raise wde\n  File \"/usr/local/lib/python3.7/site-packages/appium/webdriver/errorhandler.py\", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File \"/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1558365279203, "stop": 1558365324585}], "start": 1558365279203, "stop": 1558365325167}